From fa1b98e4eafbc4e4145328b094388328b83bb647 Mon Sep 17 00:00:00 2001
From: q66 <daniel@octaforge.org>
Date: Sat, 25 Jul 2020 17:29:23 +0200
Subject: [PATCH] mkinitrd-dracut.sh: use vmlinux regex for ppc*, vmlinuz for
 i686

Previously this would not catch ppc64le, now it does; same with
i686.
---
 mkinitrd-dracut.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mkinitrd-dracut.sh b/mkinitrd-dracut.sh
index 82d90684..c2141d68 100755
--- a/mkinitrd-dracut.sh
+++ b/mkinitrd-dracut.sh
@@ -57,10 +57,10 @@ default_kernel_images() {
         s390|s390x)
             regex='image'
             ;;
-        ppc|ppc64)
+        ppc*)
             regex='vmlinux'
             ;;
-        i386|x86_64)
+        i?86|x86_64)
             regex='vmlinuz'
             ;;
         arm*)

